tests_integration/plugins/automatic/node_modules/prettier-plugin-bar/index.js

```diff
@@ -19,5 +19,8 @@ module.exports = {
     bar: {
       print: path => concat(["bar+", path.getValue().text])
     }
+  },
+  defaultOptions: {
```

Comment:
@ikatyang mentioned this above, but could you add an integration test to ensure that this `tabWidth` is applied?